### ISSUE TITLE
Option to restart after device unplugged.

### ISSUE THREAD

**Issue by lividhen** [gdzx/audiosource#5]
It'd be great to have a launch option to keep it permanently running so you can just plug in your device and use it as a mic without any setup.

**Comment by gdzx** [comment]
Good idea. It wouldn't take long to implement but I do not have the bandwidth at the moment. Though if you are willing to make a PR for this feature I will gladly review it.

**Comment by e9x** [comment]
Just automatically run the script in the background on startup

Something like this in your startup scripts might do it.

```sh
#!/bin/bash
while sleep 0.1; do
./audiosource/audiosource run
done
```
